website/pages/en/developing/substreams-powered-subgraphs-faq.mdx

```diff
@@ -0,0 +1,89 @@
+
+## What are the benefits of using Substreams-powered subgraphs?
+
+Substreams-powered subgraphs combine all the benefits of Substreams with the queryability of subgraphs. They bring greater composability and high-performance indexing to The Graph. They also enable new data use cases; for example, once you've built your Substreams-powered Subgraph, you can reuse your [Substreams modules](/substreams/developers-guide/creating-your-manifest/#module-definitions) to output to different [sinks](/substreams/developers-guide/sink-targets/#substreams-sinks-overview) such as PostgreSQL, MongoDB, and Kafka.
+
+## What are the benefits of Substreams?
+
+There are many benefits to using Substreams, including:
+
+- Composable: You can stack Substreams modules like LEGO blocks, and build upon community modules, further refining public data.
+
+- High-performance indexing: Orders of magnitude faster indexing through large-scale clusters of parallel operations (think BigQuery).
+
+- Sink anywhere: Sink your data to anywhere you want: PostgreSQL, MongoDB, Kafka, subgraphs, flat files, Google Sheets.
+
+- Programmable: Use code to customize extraction, do transformation-time aggregations, and model your output for multiple sinks.
+
+- All the benefits of the Firehose.
+
+## What is the Firehose?
+
+Developed by [StreamingFast](https://www.streamingfast.io/), the Firehose is a blockchain data extraction layer designed from scratch to process the full history of blockchains at speeds that were previously unseen. Providing a files-based and streaming-first approach, it is a core component of StreamingFast's suite of open-source technologies and the foundation for Substreams.
+
+Go to the [documentation](https://firehose.streamingfast.io/) to learn more about the Firehose.
+
+## What are the benefits of the Firehose?
+
+There are many benefits to using Firehose, including:
+
+- Lowest latency & no polling: In a streaming-first fashion, the Firehose nodes are designed to race to push out the block data first.
+
+- Prevents downtimes: Designed from the ground up for High Availability.
+
+- Never miss a beat: The Firehose stream cursor is designed to handle forks and to continue where you left off in any condition.
+
+- Richest data model:  Best data model that includes the balance changes, the full call tree, internal transactions, logs, storage changes, gas costs, and more.
+
+- Leverages flat files: Blockchain data is extracted into flat files, the cheapest and most optimized computing resource available.
+
+## Where can devs access more information about Substreams-powered subgraphs and Substreams?
+
+The [Substreams documentation](/substreams/README/) will teach you how to do Substreams programming.
```

Comment:
"will teach you how to build Substreams modules"

Reply:
Good catch! Much better now